draft-ietf-quic-recovery.md

```diff
@@ -946,13 +946,14 @@ latest_rtt:
 
 smoothed_rtt:
 : The smoothed RTT of the connection, computed as described in
-  {{?RFC6298}}
+  Section {{smoothed-rtt}}
```

Comment:
```suggestion
  Section {{smoothed-rtt}}.
```

Reply:
I think that you mean
```suggestion
  {{smoothed-rtt}}.
```

(Section references include the string "Section ").